content/post/monthy_update_september.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Monthly Update - September"
+date: 2022-10-13T10:00:00+01:00
+tags: ["3D rendering","ASIC", "course" ]
+images: ["mouse_chip.jpg"]
+featured_image: "mouse_chip.jpg"
```

Comment:
what's the reasoning behind the mouse chip? maybe use the thumbnail I made for the video instead?